### ISSUE TITLE
NLS eReader Zoomax driver does not work with all units

### ISSUE THREAD

**Issue by florin-trutiu** [nvaccess/nvda#18406]
### Steps to reproduce:
Try to connect the device via Bluetooth or USB. 
### Actual behavior:

Some NLS eReader Zoomax units cannot be detected and cannot communicate with NVDA.
### Expected behavior:

All NLS eReader Zoomax units should be autodetected, either over USB or Bluetooth and should work with NVDA.
#### NVDA installed/portable/running from source:
The issue is present regardless of the way NVDA is run.
#### NVDA version:
NVDA 2025.2beta1
#### Windows version:
The issue appears on all Windows versions.
